x-pack/plugins/endpoint/common/types.ts

```diff
@@ -25,35 +25,39 @@ export type ImmutableObject<T> = { readonly [K in keyof T]: Immutable<T[K]> };
 export class EndpointAppConstants {
   static ALERT_INDEX_NAME = 'my-index';
   static ENDPOINT_INDEX_NAME = 'endpoint-agent*';
-}
-
-export interface AlertResultList {
+  static EVENT_INDEX_NAME = 'endpoint-events-*';
   /**
-   * The alerts restricted by page size.
+   * Phase 0 events are stored in indices with endgame-* prefix
    */
-  alerts: AlertData[];
+  static ENDGAME_INDEX_NAME = 'endgame-*';
```

Comment:
endgame? Is this the consensus now?

Reply:
`ENDGAME_INDEX_NAME` is for the events that evrelay in the SMP produces. Evrelay uses the index patter `endgame-<evrelay version>-<roll over version>`
https://github.com/elastic/endgame-evrelay/blob/dev/internal/es/create_template.go#L23

We're probably not going to change the index that evrelay streams events to. The **new** index is still TBD but the place holder I used was `endpoint-events-`

Reply:
I fine with the value of the variable it is just the name of the variable, because it could be anything and must be consi. I am not sure we want to carry over endgame in this way. But I may not have all the context or history.

Reply:
oh I see, how about `LEGACY_INDEX_NAME`? or `LEGACY_INDEX_PATTERN`?

Reply:
I always balk at `LEGACY`.  From the outside I see a versioned implementation. I also think you need the type of document stored in the document to qualify what the index holds. This way, apply a version to it further clarifies the document type.

Reply:
Not to explode the first pass implementation--but there is a plan to make the index patterns configurable, correct? If not, this is going to break anyone who wants to dynamically modify the indices where we push events to and also have the unfortunate side-effect that we'd never be able to use resolver with other data sources other than Endpoint.

I'm cool with separating the config implementation into a subsequent upcoming PR, but just want to make sure that we're all on the same page that this is just temporary...

Reply:
Yeah the configuration part is still TBD, the package manager is the one that will create the index patterns in kibana so it needs to be configured in there probably but I'm not quite sure how that's going to be done